streams/src/main/java/org/apache/kafka/streams/processor/TaskId.java

```diff
@@ -25,11 +25,13 @@
 import java.nio.ByteBuffer;
 
 /**
- * The task id representation composed as topic group id plus the assigned partition id.
+ * The task ID representation composed as topic group ID plus the assigned partition ID.
```

Comment:
Oh, the point I was trying to make: The actual implementation is, as you said, not important (and not part of the contract), thus we should not use wording that might suggest it is.  That said, I realize I might be overthinking this.  Feel free to keep the PR code as is.